src/lint/presets.ts

```diff
@@ -1,5 +1,6 @@
 import { RulesConfig } from "../config/schema";
 
+export const DEFAULT_PRESET_NAME = "default";
```

Comment:
In the project, we are using camelCase for constants, and CAPITAL_SNAKE_CASE for environment variables. Let's use camelCase for this constant.